README.md

```diff
@@ -8,6 +8,19 @@ Need help with Node.js? You've come to the right place!
 If you have a question about Node.js, [create a new issue](https://github.com/nodejs/help/issues/new)
 and ask there!
 
+When inserting you're code:
+use 3 backticks, right under the tilde & atop of the tab key followed by js: then end your code with 3 tilde. As seen below, except without the round brackets.  
+
+(```js)
```

Comment:
I guess the parentheses here are to avoid ```` ```js … ``` ```` from being rendered as markdown? You can escape backticks by using more backticks, i.e. writing ````` ```` ```js ```` ````` in markdown renders as ```` ```js ````, or indenting these lines with spaces.

Fwiw, there’s a canned reply I’ve been using here from time to time; here it is (feel free to click the edit button here to see how it looks like as raw markdown):

----------

And by the way, you can wrap pieces of source code that you post on github in  ```` ``` ```` so that it becomes more readable, for example:

    ```js
    console.log('Hello world!');
    ```

becomes

```js
console.log('Hello world!');
```

:smile: